rapid_pro_tools/rapid_pro_client.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+
+from core_data_modules.traced_data import TracedData, Metadata
+from core_data_modules.util import TimeUtils
+from temba_client.v2 import TembaClient
+
+
+class RapidProClient(object):
+    def __init__(self, server, token):
+        self.rapid_pro = TembaClient(server, token)
+        
+    def get_flow_id(self, flow_name):
+        flows = self.rapid_pro.get_flows().all(retry_on_rate_exceed=True)
+        matching_flows = [f for f in flows if f.name == flow_name]
+
+        if len(matching_flows) == 0:
+            available_flow_names = [f.name for f in flows]
+            raise KeyError(f"Requested flow not found on RapidPro (Available flows: {', '.join(available_flow_names)})")
+        if len(matching_flows) > 1:
+            raise KeyError("Non-unique flow name")
+
+        return matching_flows[0].uuid
+
+    def get_flow_ids(self, flow_names):
+        return [self.get_flow_id(name) for name in flow_names]
+
+    def get_flow_definitions_for_flow_ids(self, flow_ids):
+        return self.rapid_pro.get_definitions(flows=flow_ids, dependencies="all")
+
+    def get_raw_runs_for_flow_id(self, flow_id, range_start_inclusive=None, range_end_exclusive=None):
```

Comment:
What does 'raw' mean in this context?

Reply:
The data that comes out of Rapid Pro. I was trying to distinguish it from the traced runs which this API can also provide. Is there something I can do to be more clear?

Reply:
Give me until Monday 18th to think about this.

Reply:
I've failed to think of a better answer to this - let's go ahead with 'raw'